data/yaml/venues/knowledgenlp.yaml

```diff
@@ -0,0 +1,2 @@
+acronym: KnowledgeNLP
+name: The 3rd Workshop on Knowledge Augmented Methods for NLP
```

Comment:
The venue name should not include "The 3rd". That is a particular meeting of the venue, not the venue name itself.

Reply:
Addressed.